### PR TITLE
feat(stack): add defaults for align prop

### PR DESCRIPTION
**Pull request by ninamarina**
## Description
https://dialpad.atlassian.net/browse/DLT-1265
Adds defaults for aligning items in the Stack component.
Documentation will be updated in the `dialtone-vue` repository.

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [ ] Update, remove, or extend all affected documentation.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

### CODE REVIEW

**Review by juliodialpad — CHANGES_REQUESTED** [review]
I think we don't need to have those as utility classes.

I'd be better to just set those default values for row and column variants of the stack component.

And just say in the documentation that by default row stack will have `align-items: center` and column stack has `justify-content: flex-start` set by default.

**Comment by ninamarina** [comment]
> I think we don't need to have those as utility classes.
> 
> I'd be better to just set those default values for row and column variants of the stack component.
> 
> And just say in the documentation that by default row stack will have `align-items: center` and column stack has `justify-content: flex-start` set by default.

Changed in 3353c31b30dff7b8467815fd54323fee3038d21a

File: lib/build/less/components/stack.less [hunk 1]
```diff
@@ -17,18 +17,30 @@
 .direction-options() {
   &--column {
     --stack-direction: column;
+
+    align-items: normal;
```
Comment (iropolo) [inline lib/build/less/components/stack.less]: `normal` is the default value of `align-items` Is this adding value here?

File: lib/build/less/components/stack.less [hunk 2]
```diff
@@ -17,18 +17,30 @@
...
+    justify-content: flex-start;
   }
 
   &--row {
     --stack-direction: row;
+
+    align-items: center;
+    justify-content: normal;
```
Comment (iropolo) [inline lib/build/less/components/stack.less:34]: is this `justify-content: normal;` css property doing some effect?

**Review by iropolo — APPROVED** [review]
pre-approving it.

Have some doubt on the use of `align-items: normal` and `justify-content: normal` but if it is intended to solve some issue or edge case would be cool to review it here, if it is not I would remove them.

Thanks,

File: lib/build/less/components/stack.less (hunk 2 above)
Comment (ninamarina) [reply lib/build/less/components/stack.less:34]: It's to override the default defined here https://github.com/dialpad/dialtone/pull/961/files#diff-a6abaef1f0bdd89f14149d49806e8d9d66c282871015d46657d8ff6b988a8566R55

File: lib/build/less/components/stack.less (hunk 1 above)
Comment (ninamarina) [reply lib/build/less/components/stack.less]: Maybe for the `align-items` I don't need it, but I do for `justify-content`, so I thought for clarity and consistency add both

File: lib/build/less/components/stack.less (hunk 2 above)
Comment (iropolo) [reply lib/build/less/components/stack.less:34]: duh, right, didnt see it is under `&--row {` 👍🏼

File: lib/build/less/components/stack.less (hunk 1 above)
Comment (iropolo) [reply lib/build/less/components/stack.less]: Would remove it if it is not adding value. 
in the other hand, `justify-content` is adding something necessary to work correctly.
Up to you, looks good to me I already approved it 👍🏼

Comment (ninamarina) [reply lib/build/less/components/stack.less]: I take your suggestion, let's have the less lines of css we can 👍

**Review by braddialpad — APPROVED** [review]
Looks good!

**Review by juliodialpad — APPROVED** [review]
Looking good, thanks Nina!

I'll create a ticket to improve this component, instead of using media queries for responsive versions we should be using our dialtone generators function but that's another story. I'll create the ticket and refer back to this PR.

**Comment by juliodialpad** [comment]
Cleanup stack component ticket: https://dialpad.atlassian.net/browse/DLT-1270

**Pull request merged**